lib/workers/repository/update/pr/index.ts

```diff
@@ -255,7 +255,7 @@ export async function ensurePr(
               elapsedHours < config.prNotPendingHours))
         ) {
           logger.debug(
-            `Branch is ${elapsedHours} hours old - skipping PR creation`,
+            `Branch is ${elapsedHours} hours old - skipping PR creation as prNotPendingHours is set to ${String(config.prNotPendingHours)}`,
```

Comment:
It'd be nice if we added `prNotPendingHours` to the `config` object, so we have type information (as a follow-up)

Reply:
yeah. ts complains otherwise